FitNesseRoot/DbFit/AcceptanceTests/JavaTests/DerbyTests/FlowMode/SetUp/content.txt

```diff
@@ -1,3 +1,9 @@
 !|dbfit.EmbeddedDerbyTest|
 
 !|Connect|jdbc:derby:memory:dbfitdb;create=true|
+
+|Execute Ddl|${CREATEUSERTABLE}|
+
+|Execute Ddl|${CREATEMAKEUSERPROC}|
+
+|Commit|
```

Comment:
I'm using the `Commit` here to simulate having a permanent table in a DB where the structure (DDL) is persisted rather than created for each test.

This is to enable the tests, that include the slightly irregular `TransactionsTest`, to run properly. `TransactionsTest` includes a `Commit` and this makes commits the DDL and so the next test fails because of the `Execute Ddl` in the `SetUp`.

The `Rollback` in the `TearDown` undoes the DML that's executed in the test as would normally happen at the end of a flow mode test.

I then need to drop the DB objects and commit that drops so that any subsequent test's `SetUp` will run correctly.

It might be better to do something with the `TransactionsTest` test but I'm not sure what.